### DISCUSSION TITLE
endgame.id - privacy policy notice

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1683] [Personal Filter]
### Sample URL

https://endgame.id/

### Ad Blocker

uBlock Origin

### Keterangan

![image](https://github.com/realodix/AdBlockID/assets/19517680/f65443a3-2527-430c-b827-abdcbbca9570)

**Comment by Jokopentil** [comment]
Itu situs berbahasa inggris harusnya lapornya langsung ke ublock origin atau easylist

**Comment by 3xploiton3** [comment]
> Itu situs berbahasa inggris harusnya lapornya langsung ke ublock origin atau easylist

ini situs milik pak gita orang indonesia, domainnya aja .id itu hanya pakai plugin translate

**Comment by realodix** [comment]
Gunakan ini

```adblock
endgame.id##+js(set-cookie, endgame-accept-policy, true)
```
